src/content/migrate/5.md

```diff
@@ -144,58 +133,95 @@ Consider removing defaults:
     - This doesn't apply to the `webpack(..., callback)` form which automatically closes.
     - This is optional if you use webpack in watching mode until the user ends the process. The idle phases in watch mode will be used for this kind of work.
 
-#### Run a single build and follow advises
+### Run a single build and follow advises
 
-Please make sure to read errors/warnings carefully.
+Please make sure to read the building errors/warnings carefully. If there is no corresponding advise, please create an issue and we will try to resolve it. 
 
-If there is no corresponding advise? Please create an issue and we will try to resolve it. Repeat this step until you solved at least level 3 or 4:
+Repeat the following steps until you solved at least level 3 or 4:
 
-- Level 1: Schema validation fails. Configuration options have changed. There should be a validation error with a `BREAKING CHANGE:` note, or a hint which option should be used instead.
-- Level 2: webpack exits with an error. The error message should tell you what needs to be changed.
-- Level 3: Build Errors. The error message should have a `BREAKING CHANGE:` note.
-- Level 4: Build Warnings. The warning message should tell you what can be improved.
-- Level 5: Runtime Errors. This is tricky. You probably have to debug to find the problem. A general advise is difficult here.
-- Level 6: Deprecation Warnings. You probably get a lot of deprecation warnings. This is not directly a problem. Plugins need time to catch up with core changes. Please report these deprecations to the plugins. These deprecations are only warnings and the build will still work with only minor drawbacks (like less performance).
-- Level 7: Performance issues. Usually performance should improve with webpack 5, but there are also a few cases where performance get worse.
+- Level 1: __Schema validation fails__.
+    
+    Configuration options have changed. There should be a validation error with a `BREAKING CHANGE:` note, or a hint which option should be used instead.
+
+- Level 2: __Webpack exits with an error__.
+    
+    The error message should tell you what needs to be changed.
+
+- Level 3: __Build Errors__.
+
+    The error message should have a `BREAKING CHANGE:` note.
+
+- Level 4: __Build Warnings__.
+    
+    The warning message should tell you what can be improved.
+
+- Level 5: __Runtime Errors__.
+
+    This is tricky. You probably have to debug to find the problem. A general advise is difficult here. But we do list some common advices below regarding Runtime Errors:
 
-- Regarding Runtime Errors:
     - `process` is not defined.
         - webpack 5 does no longer include a polyfill for this Node.js variable. Avoid using it in the frontend code.
         - Want to support frontend and browser usage? Use the `exports` or `imports` package.json field to use different code depending on the environment.
```

Comment:
> frontend and browser usage

It should be server and browser, right?

Reply:
Actually it does mean `frontend and browser`, there won't be any problem using `process` server side.

Reply:
Yeah, I see it now. Then I would ask what is the difference between frontend and browser, for me browser usage is enough.

Or given the context, which is using server, we could word it differente: client usage.

Reply:
Yeah, I think `browser usage` should be enough. It's not me who wrote those texts, so I don't really know the context here.

Reply:
yeah, I know, just wanted to clarify it :)